routers/web/repo/issue.go

```diff
@@ -2044,6 +2044,17 @@ func ViewIssue(ctx *context.Context) {
 		return user_service.CanBlockUser(ctx, ctx.Doer, blocker, blockee)
 	}
 
+	canWriteProjects := ctx.Repo.Permission.CanWrite(unit.TypeProjects)
+	ctx.Data["CanWriteProjects"] = canWriteProjects
+
+	if canWriteProjects && issue.Project != nil {
```

Comment:
If the user have read permission, they should view the status.

```suggestion
	if issue.Project != nil {
```

Reply:
this option used to load chose list, not for curent status view, the curent status will be loaded by `issue.LoadAttributes()` -> `issue.LoadProjectIssue()`